pkg/arvo/app/aqua.hoon

```diff
@@ -428,17 +480,47 @@
     this
   ==
 ::
+::  Make changes to azimuth state for the current fleet
+::
+++  poke-azimuth-action
+  |=  act=azimuth-action
+  ^-  (quip card:agent:gall _state)
+  =.  this  apex-aqua  =<  abet-aqua
+  ^+  this
+  ?-  -.act
+  ::
+      %init-azimuth
+    =.  azi.piers  *az-state
+    start-azimuth-timer
+  ::
+      %spawn
+    =.  state  (spawn who.act)
+    this
+  ::
+      %breach
+    ::  should we remove the pier from state here?
+    =.  state  (breach who.act)
+    this
+  ::
+  ==
+::
 ::  Apply a list of events tagged by ship
 ::
 ++  poke-aqua-events
   |=  events=(list aqua-event)
-  ^-  (quip card:agent:gall state)
+  ^-  (quip card:agent:gall _state)
   =.  this  apex-aqua  =<  abet-aqua
   %+  turn-events  events
   |=  [ae=aqua-event thus=_this]
   =.  this  thus
   ?-  -.ae
+  ::
       %init-ship
+    ::  XX  Note that the keys that get passed in are unused. The keys field
+    ::      should be deleted now that aqua is capable of managing azimuth state
+    ::      internally. Its been left this way for now until all the ph tests
+    ::      can be rewritten
```

Comment:
To make a fake ship you just pass in keys of `~` still, right?

Reply:
Yeah, it should probably be changed to take in `fake=?` instead